server/app.js

```diff
@@ -1,16 +1,21 @@
+require('dotenv').config();
 const createError = require("http-errors");
 const express = require("express");
 const { join } = require("path");
 const cookieParser = require("cookie-parser");
 const logger = require("morgan");
+const connectDB = require('./database')
 
 const indexRouter = require("./routes/index");
 const pingRouter = require("./routes/ping");
+const userRouter = require("./routes/user")
 
 const { json, urlencoded } = express;
 
 var app = express();
 
+connectDB()
```

Comment:
forgetting a few semicolon - make sure that we have prettier installed, it would catch these